docs/guides/social-signin/overview.mdx

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
Just a nitpick - this section is now gone: https://www.ory.sh/docs/kratos/guides/sign-in-with-github-google-facebook-linkedin#configuration-from-environment-variable

Can we re-add it under "self hosted"?

Reply:
Done, now that's a reusable :::tip added to all relevant docs :)

Reply:
😎